### PR TITLE
Adds docs reference for twa-manifest.json

### PR DESCRIPTION
**Pull request by andreban**

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -273,6 +273,122 @@ bubblewrap fingerprint generateAssetLinks <flags>
...
+Android application.
+
+Developers who want to change their application configuration after running `init` can do so by
+editing this file then running the `update` command.
+
+Fields:
+
+|Name|Type|Required|Description|
+|:--:|:--:|:------:|:---------:|
+|packageId|string|true|The [application id](https://developer.android.com/studio/build/application-id) for the output Android app.|
```
Comment (PEConn) [inline packages/cli/README.md:289]: Are these in any specific order?

File: packages/cli/README.md [hunk 2]
```diff
@@ -273,6 +273,122 @@ bubblewrap fingerprint generateAssetLinks <flags>
...
+|display|`'standalone'` \| `'fullscreen'` \| `'fullscreen-sticky'`|false|The initial display mode for the Android application. Defaults to `standalone`.|
+|themeColor|string|true|The color used for the status bar.|
+|navigationColor|string|true|The color used for the navigation bar.|
+|navigationColorDark|string|false|The color used for the navigation bar when the device is in dark mode. Defaults to `#000000`.|
+|navigationDividerColor|string|false|The color used for the navigation bar divider. Defaults to `#000000`.|
+|navigationDividerColorDark|string|false|The color used for the navigation bar divider when the device is dark mode. Defaults to `#000000`.|
+|backgroundColor|string|true|Color used for the splash screen background|
+|enableNotifications|boolean|true|Set to true to enable notification delegation.|
+|startUrl|string|true|The start path for the TWA. Must be relative to the domain.|
+|iconUrl|string|true|Full URL to an the icon used for the application launcher and splash screen. Must be at least 512x512.|
```
Comment (PEConn) [inline packages/cli/README.md:302]: Probably specify 512x512 *px*

File: packages/cli/README.md [hunk 3]
```diff
@@ -273,6 +273,122 @@ bubblewrap fingerprint generateAssetLinks <flags>
...
+|maskableIconUrl|string|false|Full URL to an the icon used for maskable icons, when supported by the device.|
+|monochromeIconUrl|string|false|Full URL to a monochrome icon, used when displaying notifications.|
+|splashScreenFadeOutDuration|number|true|Duration for the splash screen fade out animation.|
+|signingKey|[SigningKeyInfo](#SigningKeyInfo)|true|Information on the signing keys used to sign the output Android application. Read the [SigningKeyInfo](#SigningKeyInfo) section for details.|
+|appVersionCode|number|false|`versionCode` for the Android application. Check the [Android docs](https://developer.android.com/studio/publish/versioning) for details.|
+|appVersion|string|false|`versionName` for the Android application. Check the [Android docs](https://developer.android.com/studio/publish/versioning) for details.|
+|shortcuts|[ShortcutInfo](#ShortcutInfo)[]|false||
```
Comment (PEConn) [inline packages/cli/README.md:309]: Just given up on this one :-P

File: packages/cli/README.md [hunk 4]
```diff
@@ -273,6 +273,122 @@ bubblewrap fingerprint generateAssetLinks <flags>
...
+|generatorApp|string|false|Identifier for tool used to generate the Android project. Bubblewrap uses `bubblewrap-cli`. Should only be modified by generator apps.|
+|webManifestUrl|string|false|Full URL to the PWA Web Manifest. Required for the application to be compatible with Chrome OS devices.|
+|fallbackType|`'customtabs'` \| `'webview'`|false|`customtabs` or `webview`|
```
Comment (PEConn) [inline packages/cli/README.md:312]: Description here could use a few more words.

File: packages/cli/README.md [hunk 5]
```diff
@@ -273,6 +273,122 @@ bubblewrap fingerprint generateAssetLinks <flags>
...
+Enables the [AppsFlyer SDK](https://support.appsflyer.com/hc/en-us/articles/360002330178-Using-AppsFlyer-with-TWA#introduction) in the Android application. Includes additional libraries and is not compatible with Chrome OS.
+
+|Name|Type|Required|Description|
+|:--:|:--:|:------:|:---------:|
+|enabled|boolean|true|Set to `true` to enable the feature.|
+|appsFlyerId|boolean|true|The appsflyer id.|
+
+#### LocationDelegationConfig
+
+Delegates the location permission dialog to the Android system, instead of showing the browser dialog. Recommended when the application requests the location permission, but not enabled by default as it includes additional dependencies in the application.
```
Comment (PEConn) [inline packages/cli/README.md:344]: Mention the binary size increase?

Comment (PEConn) [reply packages/cli/README.md:344]: Or maybe the Features table could have a binary size increase column for each feature.

Comment (andreban) [reply packages/cli/README.md:344]: Added a column.

File: packages/cli/README.md (hunk 4 above)
Comment (andreban) [reply packages/cli/README.md]: Yep, was pending that. Added.

File: packages/cli/README.md (hunk 3 above)
Comment (andreban) [reply packages/cli/README.md]: Rephrased :D

File: packages/cli/README.md (hunk 2 above)
Comment (andreban) [reply packages/cli/README.md]: Added

File: packages/cli/README.md (hunk 1 above)
Comment (andreban) [reply packages/cli/README.md]: Sorted alphabetically now.

**Review by PEConn — APPROVED** [review]

**Pull request merged**